react/wbooks/src/services/UserService.js

```diff
@@ -0,0 +1,28 @@
+import { create } from 'apisauce';
+
+const baseURL = process.env.REACT_APP_USER_BASE_URL;
+
+const api = create({
+  baseURL,
+  timeout: 15000
+});
+
+export const createUser = async ({ email, password, passwordConfirmation, name, lastname, locale }) => {
+  const response = await api.post('/users', {
+    user: {
+      email,
+      password,
+      /* eslint-disable camelcase */
+      password_confirmation: passwordConfirmation,
+      first_name: name,
+      last_name: lastname,
+      /* eslint-enable camelcase */
+      locale
```

Comment:
in order to avoid disable eslint, there is a practice called `schemas`: `normalizers` and `denormalizers`. You can use `normalizer` to cast snake case from backend to your frontend json structure, and `denormalize` to cast your frontend json to send it to backend.
you could create a folder called `schema` in the src root folder and create a file called `authSchema`. and put this logic in `registerDenormalize`.
Another option is to check this library by @damfinkel  https://github.com/damfinkel/cerealizr

There is a bunch of solutions for this.